requirements/requirements_ansible.in

```diff
@@ -50,9 +50,7 @@ netaddr
 ovirt-engine-sdk-python==4.3.0    # minimum set inside Ansible facts module requirements
 pycurl==7.43.0.1    # higher versions will not install without SSL backend specified
 # AWX usage
-pexpect==4.6.0    # same as AWX requirement
 psutil==5.4.3    # same as AWX requirement
-ptyprocess==0.5.2 # via pexpect, but needs to be pinned. Read the blame.
```

Comment:
@shanemcd @AlanCoding do we need to persist this pin?

context: https://github.com/pexpect/ptyprocess/commit/8be490695de2aead22c55928c2cf02e330fbf3dd#diff-522adf759addbd3

Reply:
Yea @wenottingham I think after speaking w/ @shanemcd, we still need to persist this pin because newer versions don't work in py2.

Reply:
runner isn't running in py2, and ansible doesn't use this.

Reply:
I agree with @wenottingham. What's in `requirements_ansible.txt` here is what matters, and we're going from 0.5.2 to no version. Whatever is in `requirements.txt` is working somehow, and isn't affected by this.

Reply:
> runner isn't running in py2

Yep, you're right.